holoviews/plotting/bokeh/element.py

```diff
@@ -1393,20 +1372,23 @@ def _setup_autorange(self):
               }}
             }}
 
-            if (y_range_name) {{
```

Comment:
Is `y_range_name` now always defined?

Reply:
It always was, it's 'default' by default. I think the old PR was simply merged before this code was fully updated.